packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -1461,6 +1461,12 @@ contract OPContractsManagerDeployer is OPContractsManagerBase {
             output.opChainProxyAdmin, address(output.systemConfigProxy), implementation.systemConfigImpl, data
         );
 
+        // If the custom gas token feature was requested, enable the custom gas token feature in the SystemConfig
+        // contract.
+        if (isDevFeatureEnabled(DevFeatures.CUSTOM_GAS_TOKEN)) {
+            output.systemConfigProxy.setFeature(Features.CUSTOM_GAS_TOKEN, true);
+        }
```

Comment:
I think this is the wrong type of feature flag check here, dev feature flags are for while a feature is in development (i.e. not ready for prod), but this is a prod-ready config feature flag that should persist. We won't be able to deploy CGT on mainnet when it's configured as a dev feature. @smartcontracts what's your recommendation here? Changing the `deploy` interface to take a `useCGT` bool feels like a hassle, is there a simpler solution?